src/partition/growfs.c

```diff
@@ -28,7 +28,10 @@
 #include <sys/types.h>
 #include <sys/vfs.h>
 
+#if HAVE_LIBCRYPTSETUP
 #include "crypt-util.h"
+#endif
```

Comment:
This `#if` and `#endif` are not needed. See crypt-util.h.

Reply:
I've seen it, that's why I thought that there is no point in including virtually empty header.
Updated as requested.